### ISSUE TITLE
gnome-photos app does not start

### ISSUE THREAD

**Issue by mrkz** [clearlinux/distribution#187]
**Describe the bug**
The gnome-photos application does not start, when running from commandline fails with a core dump

**To Reproduce**
1.  Open a terminal and type
```
$  gnome-photos

(org.gnome.Photos:3727): GLib-GIO-ERROR **: 14:45:17.841: Settings schema 'org.freedesktop.Tracker.Miner.Files' is not installed
[1]    3727 trace trap (core dumped)  gnome-photos
```

**Expected behavior**
gnome-photos application to open successfully

**Screenshots**
*N/A*

**Environment (please complete the following information):**
 - Clear Linux OS version: 24950
 - Bundles: 
```
bootloader
bootloader-extras
c-basic
containers-basic
darktable
desktop
desktop-apps
desktop-apps-extras
desktop-assets
desktop-autostart
desktop-gnomelibs
desktop-kde-apps
desktop-kde-libs
desktop-locales
dev-utils
dev-utils-dev
editors
emacs
ethtool
gimp
git
go-basic
iproute2
joe
kernel-native
koji
kvm-host
libX11client
mail-utils
mail-utils-dev
mixer
network-basic
network-basic-dev
openssh-server
openssl
os-clr-on-clr
os-core
os-core-dev
os-core-update
os-core-update-dev
os-installer
p11-kit
package-utils
perl-basic
perl-basic-dev
python2-basic
python3-basic
qt-basic
ruby-basic
shells
storage-utils
storage-utils-dev
sudo
sysadmin-basic
sysadmin-basic-dev
telemetrics
thunderbird
user-basic
vim
vlc
web-server-basic
which
wpa_supplicant
xemacs
```
**Additional context**
Missing `org.freedesktop.Tracker.Miner.Files` schema

**Comment by mrkz** [comment]
should be fixed with release containing `gnome-photos-3.30.0-10`, probably release >= 25000

**Comment by mrkz** [comment]
Clear Linux* OS release 25000 contains the fix, closing now
